Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -123,11 +121,25 @@ public func verifySnapshot<A, B>(
         return "Recorded snapshot: …\n\n\"\(snapshotFileUrl.path)\""
       }
 
-      let data = try Data(contentsOf: snapshotFileUrl)
-      let reference = snapshotting.diffing.fromData(data)
+      var allVariantsToTest = [snapshotFileUrl]
+      for c in "abcdefghijklmnopqrstuvwxyz" {
+        let url = snapshotUrl(directoryUrl: snapshotDirectoryUrl, testName: testName, identifier: identifier, pathExtension: snapshotting.pathExtension, variant: String(c))
+        if fileManager.fileExists(atPath: url.path) {
+          allVariantsToTest.append(url)
+        } else {
+          break
+        }
+      }
```

Comment:
It wasn't immediately obvious to me if/how this could be written with a `map` without testing for all files, i.e. I don't know if there's a way to break out of the map once the first file url does not exist.